src/xhr.corexhr.js

```diff
@@ -237,11 +237,7 @@ XhrShim.prototype.send = function(data) {
   } else if (data instanceof ArrayBuffer) {
     this.xhr_.send({buffer: data});
   } else if (data instanceof Blob) {
-    var reader = new FileReader();
-    reader.onload = function() {
-      this.xhr_.send({buffer: reader.result});
-    }.bind(this);
-    reader.readAsArrayBuffer(data);
+    this.xhr_.send({blob: data});
```

Comment:
Would there be any purpose to still using FileReader here in case it's *not* Firefox? Or if Blob is good enough, why not just stick with Blob even when FileReader is supported by Firefox?